cfme/configure/access_control.py

```diff
@@ -955,7 +959,9 @@ class DetailsRoleView(RoleForm):
     def is_displayed(self):
         return (
             self.accordions.accesscontrol.is_opened and
-            self.title.text == 'Role "{}"'.format(self.context['object'].name)
+            self.title.text == 'Role "{}"'.format(self.context['object'].name) and
+            (self.accordions.accesscontrol.tree.selected_item.text ==
+                self.context['object'].name)
```

Comment:
**REQUIRED**: I would probably replace `.tree.selected_item.text` with `.tree.currently_selected[-1]` as by doing `.text` you bypass widgetastic browser. `currently_selected` returns a list or None if nothing selected.

Reply:
And you could actually check the whole tree path in there to check you indeed are where you should be, but that is optional in this case.

Reply:
@mfalesni Updated to use tree.currently_selected w/ a comment to explain the use of the last item index.